setup.py

```diff
@@ -76,6 +76,7 @@
         "gym==0.21",  # Fixed version due to breaking changes in 0.22
         "numpy",
         "torch>=1.11",
+        "typing_extensions",
```

Comment:
there should a max python version here, no?

Reply:
Oh, I perhaps misunderstood, are you suggesting we change this to e.g:
```suggestion
    'typing_extensions; python_version < "3.8.0"',
```
Right now that would break things as we always import from `typing_extensions`; it would work if I also made the change @qgallouedec suggested of gating the import by Python version.

However, right now Torch unconditionally imports on `typing_extensions` and they don't seem inclined to change this, so we will install it either way: https://github.com/pytorch/vision/issues/5942

Reply:
sorry, yes i meant python version, but if pytorch do so, then don't put any version bound (otherwise it might clash with PyTorch in the future)

Reply:
I've added Python version range for consistency with [qgallouedec](https://github.com/qgallouedec)'s suggestion. I don't think this should cause any clash with PyTorch as if Python version is new enough it's as if we just don't have the requirement.